src/lib/voiceSearchHelper/index.ts

```diff
@@ -0,0 +1,104 @@
+const STATUS_INITIAL = 'initial';
+const STATUS_ASKING_PERMISSION = 'askingPermission';
+const STATUS_WAITING = 'waiting';
+const STATUS_RECOGNIZING = 'recognizing';
+const STATUS_FINISHED = 'finished';
+const STATUS_ERROR = 'error';
+
+type VoiceSearchHelperParams = {
+  onQueryChange: (query: string) => any;
+  onStateChange: () => any;
```

Comment:
This doesn't return anything.

```suggestion
  onStateChange: () => void;
```